src/SqlConverters.cs

```diff
@@ -158,6 +164,7 @@ public virtual async Task<string> BuildItemFromAttributeAsync(SqlAttribute attri
                 await connection.OpenAsync();
                 var dataTable = new DataTable();
                 adapter.Fill(dataTable);
+                this._logger.LogInformation($"{dataTable.Rows.Count} row(s) queried from database: '{connection.Database}' using Command: '{command.CommandText}'");
```

Comment:
Here you're adding single quotes but you didn't do that above. We should be consistent - and I'd probably say the quotes are unnecessary here.